src/MultipartStream.php

```diff
@@ -111,7 +111,7 @@ private function addElement(AppendStream $stream, array $element)
     /**
      * @return array
      */
-    private function createElement($name, StreamInterface $stream, $filename, array $headers)
+    private function createElement(string $name, StreamInterface $stream, ?string $filename, array $headers): array
```

Comment:
Are you sure `$filename is okay to be nullable?

Reply:
👍

Reply:
Resolving.